test/test_autograd.py

```diff
@@ -924,19 +925,20 @@ def test_indexing(self):
         y = Variable(x, requires_grad=True)
 
         def compare(x, y, idx, indexed_tensor, indexed_var):
-            indexed_var_t = indexed_var.data
+            indexed_var_t = indexed_var
```

Comment:
The goal of this code is to ensure that indexing with a Tensor or a Variable gives the same result.
If we keep this test, then we should keep the `.data` here.
@gchanan should we keep this test though?